perfrunbook/debug_hw_perf.md

```diff
@@ -136,3 +136,44 @@ Backend stalls are caused when the CPU is unable to make forward progress execut
 6. If back-end stalls due to the cache-system and memory system are the problem, the data-set size and layout needs to be optimized.
 7. Proceed to [Section 6](./optimization_recommendation.md) to view optimization recommendations for working with a large data-set causing backend stalls.
 
+### Capturing Coherent Mesh Network hardware event counters
+
+The CMN connects the CPUs to each other, to the memory controller, the I/O subsystem and provides System Level Cache.
+Its PMU counts events such as requests to SLC, DRAM (memory bandwidth), IO bus requests or coherence snoop events.
+These metrics can be used to assess an application's utilization of such system level resources and if resources are used efficiently.
```

Comment:
Lead the paragraph with a sentence more like this that describes how the PMU is useful to people rather than the chip-tech.